pubspec.lock

```diff
@@ -7,42 +7,42 @@ packages:
       name: async
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.5.0-nullsafety.1"
+    version: "2.4.2"
```

Comment:
Why every dependency here and below has its version downgraded? Is this because branch is not rebased or there is other cause?

Reply:
I had older version of flutter than someone from team who merged changes to develop before me